setup.py

```diff
@@ -25,20 +25,20 @@
     ],
     python_requires=">=3.7",
     install_requires=[
-        "flatten-dict==0.2.0",
-        "jsons==1.0.0",
+        "flatten-dict==0.4.2",
         "jupyter==1.0.0",
         "matplotlib==3.3.4",
         "pandas>=0.25.2",
-        "requests==2.27.1",
+        "requests==2.28.1",
         "statsmodels==0.12.2",
```

Comment:
Consider updating `statsmodel` as this version doesn't compile under M1 (in another task)

Reply:
https://github.com/VorTECHsa/app-issues/issues/1969